index.js

```diff
@@ -121,29 +121,26 @@ class PackageRubyBundlePlugin {
     });
   }
 
-  dockerImage() {
-    if (this.config.dockerImage)
```

Comment:
Did you intentionally remove the ability for the user to specify their own image? I think that is a valid decision, since AWS now offers the official one, there should be little reason to stray. But if we remove it, we should remove the mentioned of `dockerImage` in the README.